scripts/prepare-unity-mobile.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+### This script is used to setup Improbable's internal build machines.
+### If you don't work at Improbable, this may be interesting as a guide to what software versions we use for our
+### automation, but not much more than that.
+
+set -e -u -x -o pipefail
+
+LOG_FILE=${1:-} # Optional argument - will log to console otherwise
+
+pushd "$(dirname "$0")/../"
+    source "scripts/pinned-tools.sh"
+    source "scripts/profiling.sh"
+
+    RUN_UNITY_PATH="$(pwd)/tools/RunUnity/RunUnity.csproj"
+popd
+
+markStartOfBlock "Setting Android dependencies"
+
+pushd "tools/AndroidDependencies"
+    dotnet run -p "${RUN_UNITY_PATH}" -- \
+        -projectPath "." \
+        -batchmode \
```

Comment:
can also go for -nographics I think

Reply:
We don't do `-nographics` for the build either. Better to stay consistent.

Reply:
There's a reason for the build to be with `-nographics` as it affects lighting builds, but that's not necessary here. However it does not add much benefit so it's OK for it not to be in here.